CHANGELOG.md

```diff
@@ -72,15 +99,96 @@ All notable changes to this project will be documented in this file.
 
 ### Testing
 
+- Add tests for 'commit_pending_proposals'
+- Verify that commit operation are returning a valid welcome if any
+- Use Index trait to access conversation from Central instead of duplicate accessor
 - Use central instead of conversation
 - Fix minor clippy lints in tests
 - Apply clippy suggestions on test sources
 - Reorganize tests in conversation.rs
 - Nest conversation tests in dedicated modules
 - Verify adding a keypackage to a ConversationMember
 
+</details>
+
+This second major release focuses on expanding our platform support and featureset
+
+Platform support status:
+
+* x86_64-unknown-linux-gnu ✅
+* x86_64-apple-darwin ✅
+* x86_64-pc-windows-msvc ❌
+* armv7-linux-androideabi ✅ (⚠️)
+* aarch64-linux-android ✅ (⚠️)
+* i686-linux-android ✅ (⚠️)
+* x86_64-linux-android ✅ (⚠️)
+* aarch64-apple-ios ✅ (⚠️)
+* aarch64-apple-ios-sim ✅ (⚠️)
+* x86_64-apple-ios ✅ (⚠️)
+* wasm32-unknown-unknown ✅
+
+Note: all the platforms marked with (⚠️) above will get a round of polish for the build process & documentation in the next release.
+
+### CoreCrypto
+
+* Majorly improved documentation across all crates. Documentation for the `main` branch can be found here. The `HEAD` of this branch should only be a tagged version.
+    * This documentation is available here: <https://wireapp.github.io/core-crypto/core_crypto/>
+* Moved the codebase to `async`
+    * This was a requirement to make everything work on the WASM target, as we cannot block the JavaScript runtime without making the browsers freeze up completely
+    * As a consequence, we forked `openmls` to [wireapp/openmls](https://github.com/wireapp/openmls)
+        * Our incremental changes, including the `async` rewrite of `openmls` is located [here](https://github.com/wireapp/openmls/pull/4)
+* Added support for MLS Group Persistence, as this was preventing clients from continuing use of their joined groups (oops!)
```

Comment:
```suggestion
* **All methods creating a commit e.g. `add_clients_to_conversation` now require to call `commit_accepted` when Delivery Service responds `200 OK`. Otherwise, it might indicate there was a `409 CONFLICT`, i.e. another client sent a commit for current epoch before and got accepted. In that case, do nothing and let things get reconciled in `decrypt_message`**
* Added support for MLS Group Persistence, as this was preventing clients from continuing use of their joined groups (oops!)
```